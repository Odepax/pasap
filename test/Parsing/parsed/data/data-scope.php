<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8"/>
		<title>Document</title>
	</head>
	<body>
		<list-scope <?= \Pasap\Pasap::scope([
			'one'   => 1,
			'two'   => 2,
			'three' => 3
		]) ?> />
	</body>
</html>
