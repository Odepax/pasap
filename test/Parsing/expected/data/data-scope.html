<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8"/>
		<title>Document</title>
	</head>
	<body>
		<dl class="DataScope">
			<dt>One</dt> -- <dd>1</dd>
			<dt>Two</dt> -- <dd>2</dd>
			<dt>Three</dt> -- <dd>3</dd>
		</dl>
	</body>
</html>
